### ISSUE TITLE
Allow option to configure babelrc location for vue-jest

### ISSUE THREAD

**Issue by AtofStryker** [vuejs/vue-jest#66]
Is this currently an option? Currently running into the issue where vue-jest cannot find my .babelrc file when vue-jest's install directory is not within referencing directory. 

example:
lerna-repo
-babelrc
-node_modules -> contains-vue jest
-lerna packages
--myComponent
--jestConfiguration

inside my jestConfiguration is a reference to the workspace vue-jest:

transform: {
            '.*\\.(vue)$': "../node_modules/vue-jest"
        }

running the above fails because vue-jest cannot find my babelrc file. however, this works:

example:
lerna-repo
-babelrc
-node_modules -> contains-vue jest
-lerna packages
--myComponent
--jestConfiguration
--**babelrc**

to get around this, i am linking the parent babelrc file and removing it within npm lifecycle scripts.
version on npm is 5.6.0 and node is 8.9.4

**Comment by eddyerburgh** [comment]
Agreed. We should use a jest global to get the location, similarly to how ts-jest does it—https://github.com/kulshekhar/ts-jest/blob/master/src/preprocessor.ts#L17

You can pass a global in in the Jest config object:
```js
{
  "jest": {
    "globals": {
      "vue-jest": {
        "babelRcFile": "babelrc.json"
      }
    }
  }
} 
```

**Comment by klak-bm** [comment]
Same problem here (using yarn workspaces and would like to have only one .babelrc file to maintain).
 
I think the priority is to look for a .babelrc file the same way Babel does it :
"Babel will look for a .babelrc in the current directory of the file being transpiled. If one does not exist, it will travel up the directory tree until it finds either a .babelrc, or a package.json with a "babel": {} hash within."

Source: https://babeljs.io/docs/usage/babelrc/#lookup-behavior

**Comment by TheCycoONE** [comment]
Sorry, why are you passing a depth of 0 to findBabelConfig.sync(process.cwd(), 0). That makes it only look in the current working directory; when it should recurse down.
